Essentials/src/main/java/com/earth2me/essentials/commands/Commandnick.java

```diff
@@ -39,41 +38,41 @@ public void run(final Server server, final CommandSource sender, final String co
             throw new NotEnoughArgumentsException();
         }
         loopOfflinePlayers(server, sender, false, true, args[0], formatNickname(null, args[1]).split(" "));
-        sender.sendMessage(tl("nickChanged"));
+        sender.sendTl("nickChanged");
     }
 
     @Override
     protected void updatePlayer(final Server server, final CommandSource sender, final User target, final String[] args) throws NotEnoughArgumentsException {
         final String nick = args[0];
         if ("off".equalsIgnoreCase(nick)) {
             setNickname(server, sender, target, null);
-            target.sendMessage(tl("nickNoMore"));
+            target.sendTl("nickNoMore");
         } else if (target.getName().equalsIgnoreCase(nick)) {
             setNickname(server, sender, target, nick);
             if (!target.getDisplayName().equalsIgnoreCase(target.getDisplayName())) {
-                target.sendMessage(tl("nickNoMore"));
+                target.sendTl("nickNoMore");
             }
-            target.sendMessage(tl("nickSet", ess.getSettings().changeDisplayName() ? target.getDisplayName() : nick));
+            target.sendTl("nickSet", ess.getSettings().changeDisplayName() ? target.getDisplayName() : nick);
         } else if (nickInUse(target, nick)) {
-            throw new NotEnoughArgumentsException(tl("nickInUse"));
+            throw new NotEnoughArgumentsException(sender.tl("nickInUse"));
         } else {
             setNickname(server, sender, target, nick);
-            target.sendMessage(tl("nickSet", ess.getSettings().changeDisplayName() ? target.getDisplayName() : nick));
+            target.sendTl("nickSet", ess.getSettings().changeDisplayName() ? target.getDisplayName() : nick);
```

Comment:
Are we adding MM support to nicknames? Presumably this will be left for a subsequent PR, just want to check

Reply:
i don't know if we want to expose mini message to users or just configs, open to internal discussion